src/__tests__/components/common/radio-toggle.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import RadioToggle from '~components/common/radio-toggle'
+
+describe('Components : Common: Card', () => {
```

Comment:
Let's rename to `Components : Common : Radio toggle`

Reply:
Oops - updated.